### ISSUE TITLE
Slideshow missing next and previous image arrows

### ISSUE THREAD

**Issue by rickmgithub** [Automattic/wp-calypso#71453]
### Quick summary

On Dara if you add a slideshow block into a page set as the home page the arrows in the previous and next boxes does not appear.

<img width="816" alt="Markup 2022-12-22 at 20 44 48" src="https://user-images.githubusercontent.com/13706336/209106220-99776d0c-3ec0-4239-8a3f-b7dbec216624.png">

When viewed on mobile the forward and back options are missing all together and there is a small transparent sqaure which appears in the top right of the slideshow block 

<img width="356" alt="Markup 2022-12-22 at 20 46 59" src="https://user-images.githubusercontent.com/13706336/209106617-5f61b9a2-1dfe-44a7-a41f-35817851cf6b.png">

### Steps to reproduce

Add a slideshow block to the home page of a Dara theme site

### What you expected to happen

The slideshow to show forward and back arrows

### What actually happened

No arrows appear

### Context

5809290-zd

### Platform (Simple, Atomic, or both?)

Simple

### Theme-specific issue?

Dara

### Reproducibility

Consistent

### Severity

None

### Available workarounds?

None

**Comment by sguzovskaia** [comment]
Another report of this in 38993370-hc
Rockfield theme. The arrows in the previous and next boxes do not appear on the slideshow block added to the home page.

**Comment by filipanoscampos** [comment]
Another case here 38588040-hc and folow up here 5818718-zd-woothemes
Theme Rivington.

Shared some CSS:

```
/* Alternative arrows to overcome slideshow bug / 5818718-zd fc */

a.wp-block-jetpack-slideshow_button-next.swiper-button-next.swiper-button-white, a.wp-block-jetpack-slideshow_button-prev.swiper-button-prev.swiper-button-white{
background-size: cover;
}

a.wp-block-jetpack-slideshow_button-next.swiper-button-next.swiper-button-white:after{
content: ">";
color: #fff;
font-size: 25px;
opacity: 0.6;
}

a.wp-block-jetpack-slideshow_button-prev.swiper-button-prev.swiper-button-white:after {
content: "<";
color: #fff;
font-size: 25px;
opacity: 0.6;
}
```

**Comment by zachspears** [comment]
Able to reproduce, keeping low priority due to low impact.

**Comment by happychait** [comment]
5839955-zen

**Comment by ersatzpenguin** [comment]
Just sharing an update to the workaround CSS here that I came up with, since we ran into issues with specificity, symbol handling, and Jetpack's position in the cascade:

```
/* Alternative arrows to overcome slideshow bug | XXXXXX-HC | NM */
a.wp-block-jetpack-slideshow_button-next.swiper-button-next.swiper-button-white, a.wp-block-jetpack-slideshow_button-prev.swiper-button-prev.swiper-button-white {
	background-size: cover;
}

a.wp-block-jetpack-slideshow_button-next.swiper-button-next.swiper-button-white:after {
	content: "\003E";
	color: #fff;
	font-size: 25px;
	opacity: .6;
}

.wp-block-jetpack-slideshow a.swiper-button-next:after, .wp-block-jetpack-slideshow a.swiper-button-prev:after, .wp-block-jetpack-slideshow .swiper-container-rtl a.swiper-button-next:after, .wp-block-jetpack-slideshow .swiper-container-rtl a.swiper-button-prev:after {
	content: "\003C";
	color: #fff;
	font-size: 25px;
	opacity: .6;
}
```

**Comment by christianguerrag** [comment]
Another report in 5840729-zen with Twenty Seventeen theme. Shared CSS workaround.

**Comment by aetta** [comment]
+1 5865150-zen using Appleton. Also shared the CSS workaround!

**Comment by shameemreza** [comment]
Another report in 5868728-zen using Exford theme. Also, the slideshow pagination moves to the left, and here is the CSS workaround I used:

```
/* Move Slideshow pagination to the center | XXXXXX-HC | NM */
.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_pagination.swiper-pagination-bullets, .wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_pagination.swiper-pagination-custom {
     justify-content: center !important;
}
```

**Comment by christianguerrag** [comment]
New report in 5861865-zen using the Obsidian retired theme. Free site, so no workaround was shared.

**Comment by cardehngeh** [comment]
New report in 5880825-zen using the Mordern theme. Personal plan, so no workaround was shared to the user.

**Comment by carinapilar** [comment]
Another one on 5884756-zd-woothemes with 2020 theme.

**Comment by carolframen** [comment]
[5874934](https://href.li/?https://woothemes.zendesk.com/agent/tickets/5874934) - Zoologist, Twenty Twenty-Three and Loudness, this happened when I used the Image Compare Block on the same page. I tested other images block (gallery and regular image block) and did not trigger it. Also tested it with a paragraph block and did not trigger it.

Tested on Baskerville 2, Image Block, Gallery Block and Image Compare block, all triggered it. But paragraph block didn't.

**Comment by shameemreza** [comment]
Another one on 5884480-zen using the Hari theme. For this site, only the slideshow pagination moves to the left; the arrows are working fine. They are using the Personal plan, so no CSS workaround has been shared.

**Comment by liviopv** [comment]
- [ ] 5884704-zd-woothemes

**Comment by liviopv** [comment]
Bumping priority to high due to the prevalence and for making the block virtually unusable.

**Comment by pauljacobson** [comment]
We have another report in 5880825-zen where the site uses the Morden theme.

**Comment by obenland** [comment]
Introduced in https://github.com/Automattic/jetpack/pull/27907, props @TimBroddin for tracking that down.

CSS concatenation on Dotcom replaces relative URLs with absolute URLs in CSS `url()` functions, including for the `#maskNext` and `#maskPrev` anchors introduced above.

**Comment by davemart-in** [comment]
I submitted a PR for this here: https://github.com/Automattic/jetpack/pull/28674

**Comment by alliebbush2** [comment]
User would like to be notified once a fix is in place: 5916765-zen

**Comment by zdenys** [comment]
38726968-hc

**Comment by davemart-in** [comment]
Update: https://github.com/Automattic/jetpack/pull/28674 has been merged. It will need to wait until the next JP release and then be merged into WP.com for this issue to be fixed.

**Comment by tanjoymor** [comment]
Another case here: 5926471-zen  Personal plan so can't share CSS workaround.
Will let user know that it will be fixed in the near future.

**Comment by shameemreza** [comment]
Another one 5940342-zen using the Maywood theme. Personal plan, so no workaround was shared with the user.

**Comment by freelylovinglife** [comment]
Another Report 5957302 -ZD

**Comment by ariel-maidana** [comment]
Please notify 5957302-zd-woothemes once the fix is in place.

**Comment by marianearchimbaud** [comment]
The Pause button is also missing. or I guess it has a white background ?
This CSS here helped to display the Pause button: 

```
.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_autoplay-paused .wp-block-jetpack-slideshow_button-pause, .wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_button-play, .wp-block-jetpack-slideshow[data-autoplay="true"] .wp-block-jetpack-slideshow_button-pause {
  background-color: #cdcdcd;
}
```


The issue was reported on this conversation: 40953957-hc

**Comment by candy02058912** [comment]
The next and previous image arrows are showing.

https://github.com/Automattic/wp-calypso/issues/71453#issuecomment-1448395015 needs a fix though.

**Comment by candy02058912** [comment]
https://github.com/Automattic/wp-calypso/issues/71453#issuecomment-1448395015 is fixed by this PR https://github.com/Automattic/jetpack/pull/29330, therefore, closing the issue.

**Comment by vajrasar** [comment]
Hi everyone 👋 Just a quick note to let you all know @Automattic/dotcom-triage-pa-group have finished following up with affected users reported in this thread.

Thanks for the reports, please let us know here if you see any future recurrence of the issue.

**Comment by khristiansnyder** [comment]
It appears the prev/next buttons are fixed, but the pause button still doesn't show the correct icon in both #6032760-zen and my own tests:

![Markup 2023-03-12 at 17 42 37](https://user-images.githubusercontent.com/41035049/224584741-5fc15eda-3589-4278-9a07-933240eacb9b.png)

**Comment by davemart-in** [comment]
> The Pause button is also missing. or I guess it has a white background ?

@marianearchimbaud @candy02058912 @khristiansnyder thanks! I'll see if I can come up with a quick PR for this issue as well.

**Comment by davemart-in** [comment]
Update: After a bit of digging, it looks like this was already fixed by @candy02058912 in https://github.com/Automattic/jetpack/pull/29330 🤘

**Comment by Addison-Stavlo** [comment]
I ran into this today on support rotation: 7582837-zen - I saw an internal note pointing to this issue.

In this case, the users' theme (twenty-twenty-one) has styles that are overriding the css specificity on the block's buttons when focused.

The slideshow block delivers the next arrow via a background image:

<img width="646" alt="Screenshot 2024-01-19 at 12 39 39 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/f67919ae-8c6f-4704-8003-afb4e1bff743">

However, there are focus styles that set the background for `a` elements to white. When a user clicks "next" or "previous" buttons on the slideshow the element becomes focused, this style kicks in, and overrides the background-image with a white background:

<img width="595" alt="Screenshot 2024-01-19 at 12 41 04 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/7254b11a-df7b-4f45-a069-2b33a55dcf95">

Inspecting the stylesheet this override on focus is coming from points to the theme:

<img width="637" alt="Screenshot 2024-01-19 at 12 42 45 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/3b95df3e-4d7c-43fc-8c6b-2a502d52c792">


Potential solutions:
* Update the block's css to have more specificity and specifically designated focus styles to retain the image in this state.
* Update the theme's to not override the entire background image here. Wouldn't `background-color` instead of `background` serve the same purpose and not override `background-image`? This style seems very general and opinionated.

**Comment by davemart-in** [comment]
> Update the block's css to have more specificity and specifically designated focus styles to retain the image in this state.

@Addison-Stavlo This option stood out to me. Seems like the best way to hopefully prevent this from happening on any theme vs. playing whack-a-mole trying to fix the issue on X number of themes.

**Comment by cat-og** [comment]
📌 REPRODUCTION RESULTS
- Tested on Atomic – Replicated


📌 FINDINGS/SCREENSHOTS/VIDEO
- In Safari and Firefox, the arrows disappeared when clicked:
![Image](https://github.com/Automattic/wp-calypso/assets/86028337/70e1704b-23ca-472c-ae71-f9f428061753)

- In Chrome, when I clicked on the arrow it reacted as though I had activated a text box, with a blinking line. It was not possible to add numbers or text though. 

![Image](https://github.com/Automattic/wp-calypso/assets/86028337/4f6d52a7-ec2f-4200-aa68-9006f78ef917)



📌 ACTIONS
- Triaged

**Comment by rw-ye** [comment]
Another report: 8181619-zd-a8c

Theme: Business

**Comment by Greatdane** [comment]
Another report: 8250959-zd-a8c

**Comment by pauljacobson** [comment]
I played into 8250959-zen and had decent results with the following variation of the Custom CSS code:

```css
/* Alternative arrows to overcome slideshow bug | 8250959-ZD | NM */
a.wp-block-jetpack-slideshow_button-next.swiper-button-next.swiper-button-white,
a.wp-block-jetpack-slideshow_button-prev.swiper-button-prev.swiper-button-white {
    background-size: cover;
    background-image: none;
}

/* Next arrow */
a.wp-block-jetpack-slideshow_button-next.swiper-button-next.swiper-button-white:after {
    content: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNSIgaGVpZ2h0PSIyNCIgZmlsbD0ibm9uZSI+PG1hc2sgaWQ9ImEiIHdpZHRoPSI4IiBoZWlnaHQ9IjEyIiB4PSI4IiB5PSI2IiBtYXNrLXR5cGU9ImFscGhhIiBtYXNrVW5pdHM9InVzZXJTcGFjZU9uVXNlIj48cGF0aCBmaWxsPSIjZmZmIiBkPSJNOC41OTggMTYuNTkgMTMuMTU2IDEyIDguNTk4IDcuNDEgMTAuMDAxIDZsNS45NyA2LTUuOTcgNnoiLz48L21hc2s+PGcgbWFzaz0idXJsKCNhKSI+PHBhdGggZmlsbD0iIzAwMCIgZD0iTS4zNDQgMGgyMy44ODJ2MjRILjM0NHoiLz48L2c+PC9zdmc+);
    color: #000000 !important;
    font-size: 25px;
    opacity: .6;
}

/* Previous arrow */
.wp-block-jetpack-slideshow a.swiper-button-prev:after,
.wp-block-jetpack-slideshow .swiper-container-rtl a.swiper-button-prev:after {
    content: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNSIgaGVpZ2h0PSIyNCIgZmlsbD0ibm9uZSI+PG1hc2sgaWQ9ImEiIHdpZHRoPSI5IiBoZWlnaHQ9IjEyIiB4PSI4IiB5PSI2IiBtYXNrLXR5cGU9ImFscGhhIiBtYXNrVW5pdHM9InVzZXJTcGFjZU9uVXNlIj48cGF0aCBmaWxsPSIjZmZmIiBkPSJNMTYuMjA3IDE2LjU5IDExLjY1IDEybDQuNTU3LTQuNTlMMTQuODA0IDZsLTUuOTcgNiA1Ljk3IDZ6Ii8+PC9tYXNrPjxnIG1hc2s9InVybCgjYSkiPjxwYXRoIGZpbGw9IiMwMDAiIGQ9Ik0uNTc5IDBoMjMuODgydjI0SC41Nzl6Ii8+PC9nPjwvc3ZnPg==);
    color: #000000;
    font-size: 25px;
    opacity: .6;
}
```

**Comment by wynwin** [comment]
📌 REPRODUCTION RESULTS
Simple - Unable to replicate
Atomic - Unable to replicate

📌 ACTIONS
- Re-tested Dara and Rockfield theme, Simple/Atomic platforms, and in Chrome/Safari to confirm that this has been resolved. 
- Closing issue accordingly.
